src/content/plugins/split-chunks-plugin.mdx

```diff
@@ -165,10 +166,18 @@ Prevents exposing path info when creating names for parts splitted by maxSize.
 
 ### splitChunks.minSize
 
-`number = 20000`
+`number = 20000` `{ [index: string]: number }`
 
 Minimum size, in bytes, for a chunk to be generated.
 
+### splitChunks.minSizeReduction
+
+`number` `{ [index: string]: number }`
+
+Minimum size reduction to the whole compilation, in bytes, needed for a chunk to be generated.
+
+T> Both `splitChunks.minSizeReduction` and `splitChunks.minSize` need to be fulfilled for a chunk to be generated. You may want to reduce `minSize` when using `minSizeReduction`.
```

Comment:
I'd rather add an example instead of that last bit
```suggestion
T> Both `splitChunks.minSizeReduction` and `splitChunks.minSize` need to be fulfilled for a chunk to be generated.
```
```suggestion
T> Both `splitChunks.minSizeReduction` and `splitChunks.minSize` need to be fulfilled for a chunk to be generated. You may want to reduce `minSize` when using `minSizeReduction`.
```